vault/resource_secrets_sync_aws_destination_test.go

```diff
@@ -92,6 +98,8 @@ resource "vault_secrets_sync_aws_destination" "test" {
   access_key_id        = "%s"
   secret_access_key    = "%s"
   region               = "%s"
+  role_arn			   = "role-arn-updated"
```

Comment:
nit: formatting

Reply:
Yeah, not sure how to fix it. It looks correct on my side, and the gofmt doesn't apply any changes

Reply:
maybe an issue of tabs vs spaces?

Reply:
Ah that was it! I needed to use spaces instead. Thank you 🙏